project/src/main/webapp/script.js

```diff
@@ -73,4 +73,11 @@ function showPage() {
   document.getElementById('preloader').style.display = 'none';
   var content = document.getElementById('content')
   content.style.cssText = 'display: block; animation: fade-in 2.5s linear';
+}
+
+/** Retrieves list of topics and associated articles from the Backend Server in JSON form. */
+async function retrieveArticles(numArticles) {
+  const requestURL = '/news?num=' + numArticles;
```

Comment:
I recommend breaking them into parts and turning them into constants

Reply:
Sorry, I'm a little confused, what should I break into parts?

Reply:
The paths. : )

const BASE_URL = "https://......./";
const NEWS_PATH = "news";